### DISCUSSION TITLE
flutter_quill table

### DISCUSSION THREAD

**Discussion by Karanjit-Pintar** [singerdmx/flutter-quill#1685] [Mobile]
Hey, I was wondering if flutter_quill also let us use tables just like web quill editor let us. The recording is attached below

https://github.com/singerdmx/flutter-quill/assets/120379205/e18bb85f-01d5-49b8-909c-efad1b2fc0dd

Is it a feature that is missing in flutter_quill or there are some extra steps that needs to be done to implement it

**Comment by EchoEllet** [comment]
Thank you for the report. It has been implemented in `flutter_quill_extensions` since 9.5.0

It still needs more time before reaching a fully stable state.

**Comment by CatHood0** [comment]
> The recording is attached below

Is this video from Quill Js editor?

**Reply by EchoEllet** [reply]
> Is this video from Quill Js editor?

It could also be from [React Quill](https://www.npmjs.com/package/react-quill) which is based on it.

**Reply by CatHood0** [reply]
Thanks, i've been searching any implementation that could be better for it. I'll take a look to improve the table implementation
